products/jbrowse-react-linear-genome-view/src/JBrowseLinearGenomeView/JBrowseLinearGenomeView.tsx

```diff
@@ -1,34 +1,41 @@
+import React, { Suspense } from 'react'
 import { observer } from 'mobx-react'
-import { Instance, getEnv } from 'mobx-state-tree'
-import React from 'react'
-import createSessionModel from '../createModel/createSessionModel'
+import { getEnv } from 'mobx-state-tree'
+import { makeStyles } from '@material-ui/core'
+import { SessionModel } from '../createModel/createSessionModel'
 import ModalWidget from './ModalWidget'
 import ViewContainer from './ViewContainer'
 
-type Session = Instance<ReturnType<typeof createSessionModel>>
-function JBrowseLinearGenomeView({
-  viewState,
-}: {
-  viewState: { session: Session }
-}) {
-  const { session } = viewState
-  const { view } = session
-  const viewType = getEnv(session).pluginManager.getViewType(view.type)
-  if (!viewType) {
-    throw new Error(`unknown view type ${view.type}`)
-  }
-  const { ReactComponent } = viewType
+const useStyles = makeStyles(() => ({
+  // avoid parent styles getting into this div
+  // https://css-tricks.com/almanac/properties/a/all/
+  avoidParentStyle: {
+    all: 'initial',
+  },
+}))
 
```

Comment:
this is a nice change, but would have been good to keep this PR about a single issue

Reply:
there wasn't really any change here except to move this code into a useStyle, it is stylistic change indeed